### DISCUSSION TITLE
Dark Theme Optional

### DISCUSSION THREAD

**Discussion by blogmangit** [uBlockOrigin/uBlock-issues#1913] [General] [answered]
Dear Ublock Team, 

I am requesting to re-enable dark theme in ublock origin stable version because we can't use dev build on firefox android. Besides, make the dark theme is optional like for advanced user only if they change the uitheme flag to dark only. In this way you are not officially imposing dark theme on anyone but if advanced user might want that they can enable that from advanced user setting.

Regards.

**Comment by u-RraaLL** [comment, marked as answer]
Dark mode was always in advanced settings. It was always optional.

It was also marked as experimental and as such to not open issues about it (being incomplete, causing problems, etc.), but people ignored that part...

As for installing dev build on Android: https://www.reddit.com/r/uBlockOrigin/comments/rrvbm5/ubo_dev_build_for_firefox/
